src/synthesizer/__init__.py

```
@@ -17,15 +49,24 @@
+    "Grid",
+    "galaxy",
+    "Galaxy",
+    "check_openmp",
+    "filters",
+    "__version__",
+    "BASE_DIR",
+    "DATA_DIR",
+    "DATABASE_FILE",
+    "GRID_DIR",
+    "TEST_DATA_DIR",
+    "INSTRUMENT_DIR",
```

Pylint (3.3.7)

[error] Undefined variable name 'INSTRUMENT_DIR' in __all__

(E0603)